AdministratorAdministration+profileInstances.yaml

```diff
@@ -0,0 +1,140 @@
+    configuration:
+      value-reference: '/core-model-1-4:control-construct/logical-termination-point=aa-2-0-0-http-s-bm-000/layer-protocol=0/http-server-interface-1-0:http-server-interface-pac/http-server-interface-capability/release-number'
+
+  - profile-name: 'GenericResponseProfile'
+    uuid: 'aa-2-0-0-response-p-003'
+    capability:
+      operation-name: '/v1/inform-about-application-in-generic-representation'
+      static-field-name: 'applicationPurpose'
+      description: 'Purpose of this application'
+      datatype: 'string'
+    configuration:
+      value-reference: '/core-model-1-4:control-construct/logical-termination-point=aa-2-0-0-http-s-bm-000/layer-protocol=0/http-server-interface-1-0:http-server-interface-pac/http-server-interface-capability/application-purpose'
+
+  - profile-name: 'GenericResponseProfile'
+    uuid: 'aa-2-0-0-response-p-004'
+    capability:
+      operation-name: '/v1/inform-about-application-in-generic-representation'
+      static-field-name: 'dataUpdatePeriod'
+      description: 'Period of updating the data inside this application'
+      datatype: 'string'
+    configuration:
+      value-reference: '/core-model-1-4:control-construct/logical-termination-point=aa-2-0-0-http-s-bm-000/layer-protocol=0/http-server-interface-1-0:http-server-interface-pac/http-server-interface-capability/data-update-period'
+
+  - profile-name: 'GenericResponseProfile'
+    uuid: 'aa-2-0-0-response-p-005'
+    capability:
+      operation-name: '/v1/inform-about-application-in-generic-representation'
+      static-field-name: 'ownerName'
+      description: 'Name of the ApplicationOwner'
+      datatype: 'string'
+    configuration:
+      value-reference: '/core-model-1-4:control-construct/logical-termination-point=aa-2-0-0-http-s-bm-000/layer-protocol=0/http-server-interface-1-0:http-server-interface-pac/http-server-interface-capability/owner-name'
+
+  - profile-name: 'GenericResponseProfile'
+    uuid: 'aa-2-0-0-response-p-006'
+    capability:
+      operation-name: '/v1/inform-about-application-in-generic-representation'
+      static-field-name: 'ownerEmailAddress'
+      description: 'Email address of the ApplicationOwner'
+      datatype: 'string'
+    configuration:
+      value-reference: '/core-model-1-4:control-construct/logical-termination-point=aa-2-0-0-http-s-bm-000/layer-protocol=0/http-server-interface-1-0:http-server-interface-pac/http-server-interface-capability/owner-email-address'
+
+  - profile-name: 'GenericResponseProfile'
+    uuid: 'aa-2-0-0-response-p-007'
+    capability:
+      operation-name: '/v1/inform-about-release-history-in-generic-representation'
+      field-name-reference: '/core-model-1-4:control-construct/logical-termination-point=aa-2-0-0-http-s-bm-000/layer-protocol=0/http-server-interface-1-0:http-server-interface-pac/http-server-interface-capability/release-list=0/release-number'
+      description: 'List of registered application names and release numbers'
```

Comment:
description in line 115 and 125 (GenericResponseProfile :: /v1/inform-about-release-history-in-generic-representation :: description) to be changed to "List of release numbers and changes"